Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -100,43 +116,94 @@ protected override void OnExecute(object parameter)
                 dialog.CheckPathExists = true;
                 dialog.Multiselect = true;
                 dialog.ShowHelp = false;
-                dialog.Title = RubberduckUI.ImportCommand_OpenDialog_Title;
-                dialog.Filter = 
+                dialog.Title = DialogsTitle;
+                dialog.Filter =
                     $"{RubberduckUI.ImportCommand_OpenDialog_Filter_VBFiles} ({_filterExtensions})|{_filterExtensions}|" +
                     $"{RubberduckUI.ImportCommand_OpenDialog_Filter_AllFiles}, (*.*)|*.*";
```

Comment:
It seems that we will get an error if we try to import a file with unknown extension. Why even allow us to select any file, then?

Reply:
Because that is the behaviour already oresen. Moreover, it is standard behaviour of file dialogs to allow you to remove the filter on file extensions.